### DISCUSSION TITLE
Tailwind v3.4

### DISCUSSION THREAD

**Discussion by fernando-deka** [tailwindlabs/tailwindcss#12321] [General]
Hey! 
I would love to have the new text-balance and text-pretty on tailwind already but it seems that it is ready to be released on v3.4.
When will v3.4 be released?

Thank you!

**Comment by adamwathan** [comment]
Hey! Will be out in a couple of weeks 👍 In the mean time you can always use arbitrary properties like this:

```html
<div class="[text-wrap:balance]">
```
